tests/test_data/input_test_generate/generate_from_csv_template4.csv

```diff
@@ -1,7 +1,7 @@
-departure_time,arrival_time,vehicle_type,distance
-2021-09-16 04:22:18,2021-09-16 19:45:41,AB-OPP,5
-2021-09-17 02:59:49,2021-09-17 22:43:10,AB-OPP,8
-2021-09-18 04:51:34,2021-09-18 20:52:25,AB-OPP,10
-2021-09-16 04:23:23,2021-09-16 20:05:36,AB-OPP,4
-2021-09-17 03:37:27,2021-09-17 20:45:32,AB-OPP,12
-2021-09-18 03:50:13,2021-09-18 22:38:46,AB-OPP,1
+departure_time,arrival_time,vehicle_type,distance,connect_cs
```

Comment:
Isn't the vehicle_id missing here?

Reply:
the vehicle_id is optional. If there is no column vehicle_id it is assigned automatically via the principle "first in, fist out". A minimum standing time can be defined in the input arguments